mm2src/mm2_tests.rs

```diff
@@ -8312,3 +8312,85 @@ fn test_get_orderbook_with_same_orderbook_ticker() {
     .unwrap();
     assert!(rc.0.is_success(), "!orderbook {}", rc.1);
 }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_conf_settings_in_orderbook() {
```

Comment:
It's also worth adding a test case similar to 
https://github.com/KomodoPlatform/atomicDEX-API/blob/e7a1ce9f71c4f0b4a288745a6554a3121e83f6a4/mm2src/mm2_tests.rs#L7928-L7950
It will verify that P2P orderbook sync requests work as expected. This test also sleeps for too long time. Please make it configurable using 
https://github.com/KomodoPlatform/atomicDEX-API/blob/e7a1ce9f71c4f0b4a288745a6554a3121e83f6a4/mm2src/lp_ordermatch.rs#L2766
as an example

Reply:
Done